src/services/helpers/types.ts

```diff
@@ -28,16 +28,18 @@ export interface FactoryResolver {
 }
 
 export enum WorkspaceStatus {
-  RUNNING = 1,
-  STOPPING,
-  STOPPED,
-  STARTING,
-  PAUSED,
-  ERROR,
+  RUNNING = 'RUNNING',
+  STOPPING = 'STOPPING',
+  STOPPED = 'STOPPED',
+  STARTING = 'STARTING',
+  PAUSED = 'PAUSED',
```

Comment:
Do we really have paused for Che Workspaces, I don't think so.

Reply:
I have fixed it